app/javascript/channels/index.js

```diff
@@ -1,4 +1,2 @@
 // Import all the channels to be used by Action Cable
 //
-
-import './mediaflux_channel';
```

Comment:
This PR replaces the WebSockets/ActionCable implementation with a good old AJAX call with long polling.